modules/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
In order to avoid errors, kindly call the constructor of the parent class passing the parameters in the same order that they appear (age, name, parent_permission).